pkg/monitoring/metrics/virt-api/connection_metrics.go

```diff
@@ -61,6 +66,14 @@ var (
 		},
 		namespaceAndVMILabels,
 	)
+
+	vmiLastConnectionTimestamp = operatormetrics.NewGaugeVec(
```

Comment:
why not a Counter?

Reply:
Although the metric value can only be increased, it is not a cumulative metric. We want to set its value and not increment it. You can see in https://kubernetes.io/docs/reference/instrumentation/metrics/#list-of-stable-kubernetes-metrics for example that all the `last_timestamp_seconds` metrics are gauges. 

I get this error from the metrics name linter:
```
kubevirt_vmi_last_connection_timestamp_seconds: non-counter metric should not have "_timestamp_seconds" suffix
```
And I think we shouldn't force it.

Reply:
Opened https://github.com/kubevirt/monitoring/pull/244 to remove this restriction.